icicle/appUtils/ntt/thread_ntt.cu

```diff
@@ -211,8 +212,20 @@ public:
     }
   }
 
+  __device__ __forceinline__ void loadGlobalDataBatched(
```

Comment:
is that only for column batch? if yes, please rename to make to code clearer

Reply:
done